hdx_hapi/endpoints/get_affected_people.py

```diff
@@ -137,6 +134,14 @@ async def get_humanitarian_needs(
     return transform_result_to_csv_stream_if_requested(result, output_format, HumanitarianNeedsResponse)
 
 
+get_humanitarian_needs.__doc__ = (
+    "UNOCHA's Humanitarian Needs data, based on the Joint and Intersectoral Analysis Framework (JIAF), "
+    'provides information about the number of people in need during a crisis.'
+    f'See the more detailed technical <a href="{CONFIG.HAPI_READTHEDOCS_OVERVIEW_URL}data_usage_guides/'
+    'affected_people/#humanitarian-needs">HDX HAPI documentation</a>, '
+    'and the <a href="https://www.jiaf.info/">original JIAF source</a> website.'
```

Comment:
It might be better to link to HPC Tools: https://api.hpc.tools/, as JIAF is not really the source, it's just a methodology